graylog2-web-interface/src/routing/__snapshots__/AppErrorBoundary.test.jsx.snap

```diff
@@ -0,0 +1,134 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
It looks like the snapshot needs some updating after the last changes.

Reply:
✅